addons/theme_default/models/theme_default.py

```diff
@@ -5,3 +5,11 @@ class ThemeDefault(models.AbstractModel):
 
     def _theme_default_post_copy(self, mod):
         self.disable_view('website_theme_install.customize_modal')
+        # Reset all fonts when switching themes
+        values = {
+            'font-number': 1,
+            'headings-font-number': 1,
+            'buttons-font-number': 1,
+            'navbar-font-number': 1,
```

Comment:
Should not you reset to `null` ? Especially in other themes, otherwise it will always be forced to 1...
Note: good catch to do it for theme_default too and not only theme_common as it was before. In fact to go deeper, all of this should be done in the main _post_copy method: disabling the customize modal view and resetting fonts.

Reply:
Why reset to `null` ? It will crash my scss :/ or I don't understand your idea ?

Reply:
Plus, should I keep theme_common.py and theme_default.py files given that they are both empty now ?

**theme_common.py**
```
from odoo import models

class ThemeCommon(models.AbstractModel):
    _inherit = 'theme.utils'
```

**theme_default.py**
```
from odoo import models

class ThemeDefault(models.AbstractModel):
    _inherit = 'theme.utils'
```

Reply:
- If they are empty remove them (and adapt __ini__.py files accordingly).
- You want to reset to what the themes defines, not force back to font 1 as if the user chose it. It cannot work like. You install a theme, it is forced to 1, then the theme says it is 5 by default and the scss don't care because there is a custo which  forces it to 1 ?

Reply:
`You want to reset to what the themes defines, not force back to font 1 as if the user chose it. It cannot work like. You install a theme, it is forced to 1, then the theme says it is 5 by default and the scss don't care because there is a custo which forces it to 1 ?`

Yes, I am aware of that and you're right but I can't put `null` as a value in this file, it will crash. I think the best thing to do is to remove those keys from the file, what do you think ?

Reply:
I'll check myself, don't touch the branch no more